docs/components/renderer.md

```diff
@@ -16,7 +16,9 @@ The `renderer` component configures a scene's
 <a-scene renderer="antialias: true;
                    gammaOutput: true;
                    sortObjects: true;
-                   physicallyCorrectLights: true;"></a-scene>
+                   physicallyCorrectLights: true;
+                   maxCanvasWidth: 1920;
+                   maxCanvasHeight: 1920;"></a-scene>
```

Comment:
1080 would be better default

Reply:
Can do – but I suppose in most instances you'd be best having them the same value thereby capping for both landscape and portrait aspect ratios